#!/bin/sh /usr/share/dpatch/dpatch-run
## 319-magic-add-pe5.dpatch by Simon Horman <horms@debian.org>
##
## DP: New magic for the Ulead Photo Explorer5 image format (Closes: #339397).

@DPATCH@

diff -Naurp file.orig/magic/Magdir/images file/magic/Magdir/images
--- file.orig/magic/Magdir/images	2008-07-02 15:16:52.000000000 +0000
+++ file/magic/Magdir/images	2008-07-15 13:39:37.000000000 +0000
@@ -600,3 +600,8 @@
 # Wavelet Scalar Quantization format used in gray-scale fingerprint images
 # From Tano M Fotang <mfotang@quanteq.com>
 0	string	\xff\xa0\xff\xa8\x00	Wavelet Scalar Quantization image data
+
+# Type:	Ulead Photo Explorer5 (.pe5)
+# URL:	http://www.jisyo.com/cgibin/view.cgi?EXT=pe5 (Japanese)
+# From:	Simon Horman <horms@debian.org>
+0	string	IIO2H	Ulead Photo Explorer5
